lua/weapons/gmod_tool/stools/starfall_screen.lua

```diff
@@ -1,4 +1,5 @@
-TOOL.Category		= "Wire - Display"
+TOOL.Category		= "Visuals/Screens"
+TOOL.Wire_MultiCategories = { "Chips, Gates" } -- Also add this to the chips, gates category so that it can be found near the processor
```

Comment:
Shouldn't this be `{ "Chips", "Gates" }` instead?
Also, why is the processor so different from the screen?